### ISSUE TITLE
Error: YAML parse error on milvus/templates/writable-deployment.yaml: error converting YAML to JSON

### ISSUE THREAD

**Issue by ghiyastfarisi** [milvus-io/milvus-helm#158]
I found this error when trying to enable milvus cluster with helm.
 
Below how the installation goes with it config: 

```
helm install                                        \
  --set cluster.enabled=true                        \
  --set persistence.enabled=true                    \
  --set mysql.enabled=false                         \
  --set mishards.replica=3                          \
  --set readonly.replica=3                          \
  --set externalMysql.enable=true                   \
  --set externalMysql.ip=192.168.99.99              \
  --set externalMysql.port=3306                     \
  --set externalMysql.user=root                     \
  --set externalMysql.password=example              \
  --set externalMysql.database=db_milvus_cluster    \
  milvus-release milvus/milvus
```

After some trial it turns out that the error comes from `--set mysql.enabled=false` configuration, and I need to disable internal mysql pods, because I already set up external mysql service.

Error message:
```
Error: YAML parse error on milvus/templates/writable-deployment.yaml: error converting YAML to JSON: yaml: line 43: did not find expected '-' indicator
```

**Comment by ghiyastfarisi** [comment]
I think I found the problem, after reading the deployment-writeable.yml file.

Found that mishards install tutorial written here https://milvus.io/docs/v0.10.5/mishards.md has some error on config name.

![alt text](https://i.postimg.cc/1R2mkBNB/typo-docs.png)

`--set externalMysql.enable=true` should be `--set externalMysql.enabled=true` 

also

`--set mishards.replica=2` and  `--set readonly.replica=2` the replica is not plural.

it is better to read directly from helm charts docs here:
https://artifacthub.io/packages/helm/milvus/milvus
